src/libServer/LookupServer.cpp

```diff
@@ -897,8 +904,11 @@ Json::Value LookupServer::GetSmartContractState(const string& address,
 
   try {
     Address addr{ToBase16AddrHelper(address)};
-    shared_lock<shared_timed_mutex> lock(
+
+    AccountStore::GetInstance().IncrementPrimaryWriteAccessCount();
```

Comment:
`Get` functions are not expected to write anything but read only. Not sure why we are incrementing WriteAccessCount here.

Reply:
`DeserializeDelta` will be incrementing it in the new commit